src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -116,7 +117,7 @@ class HoldConfirmation extends React.Component {
 
     return (
       <div id="mainContent">
-        <div className="page-header">
+        <div className="nypl-request-page-header">
```

Comment:
the HP hero style / component should only be used on the HP. In the example pages you will see the other two variants. Sorry this piece wasnt sorted out before the demo.
the following all have different treatments:
1. Home page https://nypl.github.io/design-toolkit/discovery-home.html
2. Search Results / Item page: https://nypl.github.io/design-toolkit/discovery-search.html
3. Request Screens: https://nypl.github.io/design-toolkit/discovery-request.html
If I need to rework DOM & `classes` let me know